tests/unit/component/component-that-uses-modal-dialog-test.js

```diff
@@ -0,0 +1,42 @@
+import { test, moduleForComponent } from 'ember-qunit';
+/* global Ember */
+
+moduleForComponent('component-that-uses-modal-dialog', {
+  /*
+  * Setting up the container for the modal dialog to wormhole into and
+  * injecting the id of that element into the modal-dialog component
+  */
+  beforeEach: function(){
+    var modalContainerEl = document.createElement("div");
+    modalContainerEl.id = 'modal-overlays';
+    var rootEl = document.querySelector("#ember-testing");
+    rootEl.appendChild(modalContainerEl);
+    this.container.register('config:modals-container-id', 'modal-overlays', {instantiate: false});
```

Comment:
during migration ember app from 1.13 to 2.13 found unit test with similar code. it was probably copied from related discussion. could you please provide details is this test code still relevant and how I can do that in Ember 2.0